js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadSession.tsx

```diff
@@ -573,7 +584,10 @@ const LaunchpadSession = (props: LaunchpadSessionProps) => {
   }
 
   let launchButtonWarning: string | undefined;
-  if (partitionSets.results.length && isMissingPartition(currentSession.base)) {
+  if (
+    (partitionSets.results.length || partitionSetDetails.doesAnyAssetHavePartitions) &&
```

Comment:
This is the fix! Lifting the data fetching up out of ConfigEditorConfigPicker into a hook is just so that we can reference `doesAnyAssetHavePartitions` here.